tests/interfaces/rpc/tests/getVersion.json

```diff
@@ -0,0 +1,30 @@
+{
+  "title": "getVersion",
+  "schema": {
+    "$ref": "../schemas/getVersion.json"
+  },
+  "tests": [
+    {
+      "title": "getVersion success if enabled",
+      "request": {
+        "method": "getVersion",
+        "params": []
+      },
+      "expectedResponse": {
+        "result": {
+          "softwareVersion": "0.20"
```

Comment:
why here is `"0.20"`?

Reply:
There is no specified version, it just a sample. If the software upgrade, the test needs to be changed.
Interface testing does not rely on specific versions.

Reply:
ok